src/modules/mc_pos_control/MulticopterPositionControl.cpp

```diff
@@ -88,6 +100,35 @@ int MulticopterPositionControl::parameters_update(bool force)
 		ModuleParams::updateParams();
 		SuperBlock::updateParams();
 
+		if (_param_sys_vehicle_resp.get() >= 0.f) {
+			// make it less sensitive at the lower end
+			float responsiveness = _param_sys_vehicle_resp.get() * _param_sys_vehicle_resp.get();
+
+			int num_changed = 0;
+			num_changed += paramSet(_param_mpc_acc_hor, math::lerp(1.f, 15.f, responsiveness));
+			num_changed += paramSet(_param_mpc_acc_hor_max, math::lerp(1.f, 15.f, responsiveness));
+			num_changed += paramSet(_param_mpc_man_y_max, math::lerp(50.f, 300.f, responsiveness));
```

Comment:
I would set the maximum and minimum of the parameters to what we find works best here.

Also autonomous acceleration needs to be much lower to have a percieved smooth flight because the trajectory is predictable and there's no need for stick responsiveness.

It would be cool if we set it up in a way where the default responsiveness leads to the default parameters and when adjusting the responsiveness you start from there without a step.

Reply:
I use a negative number to control the default, and 0.4 matches closely what we currently have:
```
x + MPC_ACC_DOWN_MAX [385,672] : 3.0720 (vs 3)
x + MPC_ACC_HOR [386,673] : 3.2400  (vs 3)
x + MPC_ACC_HOR_MAX [387,674] : 4.0800 (vs 5)
x + MPC_ACC_UP_MAX [388,675] : 3.2400 (vs 4)
x + MPC_JERK_AUTO [393,680] : 4.8400 (vs 4)
x + MPC_JERK_MAX [394,681] : 9.6800 (vs 8)
x + MPC_MAN_Y_MAX [402,689] : 139.2000 (vs 150)
x + MPC_MAN_Y_TAU [403,690] : 0.3667 (vs 0.08)
```